db/migrate/20240507004300_add_user_id_to_medical_records.rb

```diff
@@ -0,0 +1,10 @@
+class AddUserIdToMedicalRecords < ActiveRecord::Migration[7.1]
+  def change
+    add_reference :medical_records, :usuario, null: true, foreign_key: true, default: 1
```

Comment:
This will work fine in your local environment but you don't know if there is a user with ID 1 in other environments like testing or production.
So you need to fin or create the user.
```suggestion
    user = User.first || User.create(email: "temporary_email@example.com", password: "pass", password_confirmation: "pass")
    add_reference :medical_records, :usuario, null: true, foreign_key: true, default: user.id
```

Reply:
To do that, is it better to undo the migration and do it again? or modify my file and run the same migration?

Reply:
Given this change won't make any difference in your local, you can just commit the change so it will run for your production and testing environments. Only for this case scenario.